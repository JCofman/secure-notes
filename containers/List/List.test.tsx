import React from 'react';
import { render } from '@testing-library/react';

import List from './List';

test(`List is rendered properly`, () => {
    const { container } = render(<List />);
    expect(container).toMatchSnapshot();
});
