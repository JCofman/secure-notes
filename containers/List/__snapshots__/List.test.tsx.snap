// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List is rendered properly 1`] = `
<div>
  <ul
    class="list"
  />
</div>
`;
